src/main/java/io/github/thebusybiscuit/slimefun4/core/services/profiler/SlimefunProfiler.java

```diff
@@ -217,6 +221,11 @@ private void finishReport() {
 
         totalElapsedTime = timings.values().stream().mapToLong(Long::longValue).sum();
 
+        // We log how many milliseconds have been ticked, and how many ticks have passed
+        // This is so when bStats requests the average timings, they're super quick to figure out
+        totalMsTicked.addAndGet((long) (totalElapsedTime / 1e6));
```

Comment:
`1e6`. God I hate scientific notation.

This is a conversion from nanos to millis, right?
Can't we please use something like `TimeUnit.NANOSECONDS.convert(totalElapsedTime, TimeUnit.MILLISECONDS)` then?